### PR TITLE
doc: add basic faq documentation

### PR DESCRIPTION
**Pull request by Kaian**
This is only a basic initial version, feel free to add any missing point or remove what you don't find useful

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+IvozProvider is a provider oriented multilevel IP telephony solution exposed to the public network.
+
+### Is IvozProvider free?
+
+Yes. This project is free as in freedom, but not as in beer.
+Source code is licensed under GPLv3 or later, so you can download, modify and distribute it for free.
+
+### I would like to try, where should I start?
+
+We provide an standalone iso to test the solution in a single machine. These environments are not
```
Comment (Kaian) [inline FAQ.md]: s/an/a/g

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+We don't provide any kind of free support for installations. If you have trouble, you can contact or sales department
+at comercial@irontec.com
+
+If by any chance you receive support in Github or #ivozprovider freenode IRC channel, it's most probably provided
+during developers spare time, so be thankful for the people giving part of their time.
+
+### I have found a bug
+
+If you have a bug or something that you can prove it's not working as it should, don't hesitate to open an issue
+in Github project. We appreciate well documented bug reports that improve that help to improve the software quality.
```
Comment (Kaian) [inline FAQ.md]: improve or help to improve :sweat_smile:

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+### I have a feature request
+
+If you have a feature request that you think it would be helpful for others, don't hesitate to open an issue in github
+project. Take into account that we're a small development team and features won't be scheduled unless someone pays
+for it. If you're interested in sponsoring a feature, please contact with our sales department at comercial@irontec.com
+Otherwise, feature won't be implemented unless one of our clients find it useful.
```
Comment (Kaian) [inline FAQ.md]: s/find/finds/

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+public so everyone can see what's exactly being done during each process. But this software is shipped without any kind
+of warranty and project members have no dept with others but their clients, so avoid any kind of order or imposition.
+
+If solution doesn't fulfill your requirements or you find a better solution that fits better, use it. If you want are
+interested in improving the software, consider sponsoring any missing feature.
+
+### I would like to contribute
+
+Free projects are usually managed by small developers teams, so we highly appreciate all contributions. There are all
+kind of different task you can contribute, from coding to documenting or translating.
```
Comment (Kaian) [inline FAQ.md]: s/task/tasks/

File: FAQ.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+Current stable release is called oasis. Stable releases have frozen features, so they won't evolve too much in time.
+This provides the required stability that telephony solutions demands.
+
+If you're interested in testing newest features, you can use artemis release. Take into account that artemis release is
+not yet considered stabled and bugs can arise, so use it at your own risk.
+
+### Can I use IvozProvider without public IP address?
+
+No. Current design assume you have at least one public IP address.
```
Comment (mmadariaga) [inline FAQ.md]: assumes

File: FAQ.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Currently there is no documentation for distributed installs. This requires quite some time and we are a small team with
+lots of tasks scheduled. If you need advice or support for distributed installs, contact our sales department at
+comercial@irontec.com so we can schedule it properly.
+
+### This solution is not useful without this feature
+
+We do free (as in freedom) software because we think it's the best way to produce high quality products. Our code is
+public so everyone can see what's exactly being done during each process. But this software is shipped without any kind
+of warranty and project members have no dept with others but their clients, so avoid any kind of order or imposition.
```
Comment (mmadariaga) [inline FAQ.md]: debt?

File: FAQ.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+If solution doesn't fulfill your requirements or you find a better solution that fits better, use it. If you want are
```
Comment (mmadariaga) [inline FAQ.md]: a <strike>better</strike> solution that fits better

Comment (mmadariaga) [reply FAQ.md]: <strike>want</strike>

**Comment by Kaian** [comment]
Updated with requested changes. Thanks!

**Pull request merged**